pulser-simulation/pulser_simulation/simconfig.py

```diff
@@ -103,6 +104,8 @@ class SimConfig:
             pulses.
         amp_sigma: Dictates the fluctuations in amplitude as a standard
             deviation of a normal distribution centered in 1.
+        detuning_sigma: Dictates the fluctuations in detuning as a standard
+            deviation of a normal distribution centered in 0.
```

Comment:
Side note: it's about time we stop carrying around `SimConfig`...

Reply:
I'll take your suggestions on what to do with this, then. I agree that we need to get rid of it eventually, but what would be a good strategy for getting there? Don't people still depend on this? Is it deprecated yet?

Reply:
Not for this PR, but the idea would be to deprecate direct access to `QutipEmulator` and `SimConfig` and enforce usage of `QutipBackendV2` and `NoiseModel`